### PR TITLE
Make it compile with latest Idris 2 master

### PR DESCRIPTION
**Pull request by bamboo**
idris-lang/Idris2@1db440d3cf661384b834b088667992dffb9f6fa3

### CODE REVIEW

File: src/Server/Main.idr [hunk 1]
```diff
@@ -98,7 +98,7 @@ runServer = do
   catch (processMessage method msg)
         (\err => do
           logString Error (show err)
-          resetContext)
+          resetContext "")
```
Comment (ohad) [inline src/Server/Main.idr:101]: I think the convention is "(interactive)", but these aren't well documented, so I could be wrong. There are other places like this elsewhere, I'll just stick "(interactive)" there.

File: src/Server/ProcessMessage.idr [hunk 2]
```diff
@@ -140,7 +140,7 @@ processMessage TextDocumentDidOpen msg@(MkNotificationMessage TextDocumentDidOpe
     update LSPConf (record {openFile = Just (params.textDocument.uri, params.textDocument.version)})
     -- TODO: the compiler works directly with the file, we should upstream some changes to allow it to work with the
     --       source passed by the client.
-    resetContext
+    resetContext ""
```
Comment (ohad) [inline src/Server/ProcessMessage.idr:143]: "(interactive)"

File: src/Server/ProcessMessage.idr [hunk 3]
```diff
@@ -156,7 +156,7 @@ processMessage TextDocumentDidSave msg@(MkNotificationMessage TextDocumentDidSav
   whenNotShutdown $ whenInitialized $ \conf => do
     update LSPConf (record {openFile = Just (params.textDocument.uri, 0)})
     -- TODO: same issue of TextDocumentDidOpen
-    resetContext
+    resetContext ""
```
Comment (ohad) [inline src/Server/ProcessMessage.idr:159]: "(interactive)"

**Comment by ShinKage** [comment]
Updated to https://github.com/idris-lang/Idris2/commit/364eaff6ac2ea8a5dd0692527e16a9c2f8e8ca99

**Pull request merged**